Add Community Contributors 2020 article (#5091)
ppy/osu-wiki: news/2021-03-18-community-contributors-2020.md

---
layout: post
title: "Community Contributors: 2020"
date: 2021-03-18 10:30:00 +0000
---

Today, we celebrate some of the people who make the osu! community truly tick by awarding them our highest honour: the badge of a Community Contributor.

> ![](/wiki/shared/news/2021-03-18-community-contributors-2020/title.jpg)

Reviewer: Didn't see it since I use light theme, definitely needs a fix since osu site is darkness.

> ![](/wiki/shared/news/2021-03-18-community-contributors-2020/title.jpg)

Reviewer: Didn't notice it since I use light theme, definitely needs a fix since osu is darkness

> ![](/wiki/shared/news/2021-03-18-community-contributors-2020/title.jpg)

Author: this should be fixed now

osu! is one of the rare bastions still left in the modern-day internet where community is everything. From day one, we've been built up by the tireless efforts of hundreds of individuals who selflessly dedicate themselves towards making things more awesome for everyone.

It is by their efforts that the osu! of today exists, and we thank them by periodically seeking out the exemplars among us and granting them the Community Contributor badge, a signet of their enduring dedication and commitment to the osu! community.

![Community Contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg)

Today, we welcome more into the fold, and perhaps some that should have been welcomed long ago.

We present to you, the latest of the greatest:

## Contributors

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/hallowatcher.jpg)](https://osu.ppy.sh/users/1874761)

> **hallowatcher** has had a long and storied history with osu!, but is most renowned for his creation of the now-ubiquitous 'pickems' games that accompany the official World Cups and a number of community tournaments as well. Pickems have since become something of a sensation, attracting well over 5000+ unique users and over 40 variants hosted, and several have even been the recipients of official World Cup profile badges, all while remaining a top tier tournament competitor himself.

Reviewer 2: > several have even been the recipients of official World Cup profile badges

if this refers to the "5000+ unique users", switch it with "40 variants hosted". currently sounds like the pickems receive wc badges

> all while remaining a top tier tournament competitor himself.

i assume this is about hallowatcher, but the sentence is about pickems so it reads super weird

Author: jostled the wording around a bit

His efforts have quite literally, helped shape a scene for years to come and brought joy to tens of thousands of people. Belated though it may be, we finally welcome him aboard as a Community Contributor, and a most deserved one at that.

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/mangomizer.jpg)](https://osu.ppy.sh/users/1893718)

When he isn't setting improbable feats of calculation and endurance as the self-titled "osu!taiko masochist", **mangomizer** is an ever-present aid in the World Cup scene, either hosting in or assisting with an absolute glut of projects intended to help brighten up people's day.

A five year plus referee in the osu!taiko World Cup and the cryptic mastermind of many a World Cup bingo or the very aptly named osu! Trivium Quiz, **mangomizer** has without a doubt left a profoundly positive impact upon the community with his efforts, and stands starkly as a rare example of someone who seeks to build with and uplift others, embodying the core osu! spirit of "making things more awesome for everyone".

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/lasse.jpg)](https://osu.ppy.sh/users/896613)

> Though recently retired from the role, during his heyday **Lasse** remained consistently one of the most active members of the Nomination Assessment Team who still attended to the general duties of a normal Beatmap Nominator, advancing to stand at the very pinnacle of the overall kudosu ranking and consistently scoring a top 5 place among the BN in the process. 

Reviewer 2: > ...scoring a top 5 place among the BN in the process. 

top 5 on what leaderboard? there are no bn rankings?

Author: it references the elite candidacy thing, but i'll juggle that around

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/rockroller.jpg)](https://osu.ppy.sh/users/8388854)

Heading efforts to help guide the skinning community on osu! for many years, **RockRoller** is a seasoned veteran when it comes to all things involving anime girls and fancy shapes inside your osu! client. 

A relatively recent addition to the Global Moderation Team, **RockRoller** almost immediately distinguished himself by wading knee deep into not only several internal efforts to bring our moderation documentation up to date, but also in the development and design of several features that ultimately saved hundreds of manhours across the entire moderation team.

It is clear to any with eyes that his drive to assist in osu!'s continued development is not only skin deep.

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/xexxar.jpg)](https://osu.ppy.sh/users/2773526)

Formerly known as **Xexxar**, *I Must Decrease* has been almost solely responsible for several of the recent and long-spanning adjustments to the performance points and star rating algorithm.

> **Xexxar's** efforts have been quite literally keeping the flame alive for hundreds of thousands of progression-craving players who desire change and challenge in their climb up osu!’s global ranking ladder. The depth and complexity of this work cannot be understated - involving vagaries of math and analysis enough to make a lay person’s head spin, along with constant tweaking, sampling and experimentation to ensure things go exactly where they need to be.

Reviewer 2: ```suggestion
**Xexxar**'s efforts have been quite literally keeping the flame alive for hundreds of thousands of progression-craving players who desire change and challenge in their climb up osu!’s global ranking ladder. The depth and complexity of this work cannot be understated - involving vagaries of math and analysis enough to make a lay person’s head spin, along with constant tweaking, sampling and experimentation to ensure things go exactly where they need to be.
```

He has moonlighted as a commentator in nearly a dozen events, and continues to demonstrate an exemplary dedication towards osu!’s tournament scene in all of its forms.

Short and sweet though his blurb here may be, his contributions to the scene go beyond description in mere numbers alone, and it is with gratitude that we lift him up to take his rightful place next to the other Mega Referees(™) who have proven so integral to the community tournament scene at large.

## Paragons

Paragons represent the handful of era-defining people that shape the course of a particular game mode. They have gone so far above and beyond the call of duty that they are beacons unto themselves, examples that dictate the future paths a given mode can take, and each and every single one has been utterly instrumental to the development of their mode as a whole.

Until today, osu! remained the last game mode to not formally acknowledge any such people. No longer shall this be the case.

Now, we welcome the two new osu! Paragons:

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/pishifat.jpg)](https://osu.ppy.sh/users/3178418)

> Perhaps the first person to ever appear twice in the Community Contributor series, **pishifat’s** contributions towards osu! are so widespread and numerous at this point that they are almost difficult to count. 

Reviewer 2: ```suggestion
Perhaps the first person to ever appear twice in the Community Contributor series, **pishifat**'s contributions towards osu! are so widespread and numerous at this point that they are almost difficult to count. 
```

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/baar_vendel.jpg)](https://osu.ppy.sh/users/8679346)

Despite only entering the fray less than a year ago, **BaAR_Vendel** has proven themselves to be utterly exceptional at their role.

They have tirelessly helped numerous mappers earn their first ranked map and assisted multitudes more in getting the fabled second set of eyes needed to give a map that extra push through the process, earning themselves a strong reputation as a selfless individual who deeply cares about the mapping and modding community at large.

Very befitting of an Elite Nominator, in our eyes.

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/milan.jpg)](https://osu.ppy.sh/users/1052994)

While perhaps not the most active name on the list so far, **Milan-** is certainly one of the most consistent. They have focused on helping as many unique mappers as possible with special attention given to those new to the scene, all without compromising on the quality of the maps that they push forward into Ranked - especially notable as they've received nearly no disqualifications despite a *very* packed 2020.

> [![](/wiki/shared/news/2021-03-18-community-contributors-2020/eiri.jpg)](https://osu.ppy.sh/users/3388410)

Reviewer: Update with again version

Author: applied

> Living in two worlds, **eiri-** is perhaps one of the most interconnected hybrid beatmap nominators there has ever been. Proficient in not one but *two* game modes, **eiri-** has pulling twice their weight in pushing over 80 osu!taiko beatmaps in 2020 alone, not to mention the fact that they were an Elite Nominator recipient last year as well... for osu!.

Reviewer 2: weird to go on about how they're a proficient osu+taiko nominator, then only list the # of maps they pushed for taiko. would include the osu stat too

Author: updated wording for this

This year, they join the ranks in not one, but *two* modes (osu! and osu!taiko), technically making them the first triple recipient of the title.

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/raiden.jpg)](https://osu.ppy.sh/users/2239480)

> A recent NAT retiree, **Raiden** is indisputably a household name in the osu!taiko scene, joining a host of individuals who have managed to maintain exemplary nominator activity in their given modes while tending to the difficulties of NAT life.

Reviewer: ```suggestion
A recent BN+GMT retiree and previous long-time QAT member, **Raiden** is indisputably a household name in the osu!taiko scene, joining a host of individuals who have managed to maintain exemplary nominator activity in their given modes while tending to the difficulties of GMT life.
```

2020 saw him push more than 120 different osu!taiko beatmaps towards ranked, without a single severe disqualification, showcasing his enormous skill in identifying issues and working with mappers to get them fixed.

While he has since retired from the nominator life in general, we applaud **Raiden**'s pursuit of excellence, and wish him all the best in his endeavours elsewhere.

> [![](/wiki/shared/news/2021-03-18-community-contributors-2020/jemzuu.jpg)](https://osu.ppy.sh/users/7890134)

Reviewer: Update with again version

Author: applied x2

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/tochi.jpg)](https://osu.ppy.sh/users/3664366)

<div align="center">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/community-contributors-2020/tochi_showcase.mp4" type="video/mp4" preload="none">
    </video>
</div>

Long sporting an extremely impressive portfolio of storyboards, ranging from simple enhancements of a beatmap's background to all-out compliments to both song and beatmap, **-Tochi**'s name is likely very familiar to anyone who has played a super high production value beatmap in recent years.

Undeterred by the creative challenge of marathon length maps, they have risen to the occasion with creations like [\(S\).0ngs//---::compilation.\[TQR-f3\]](https://osu.ppy.sh/beatmapsets/1124097), which features a stunning 9 minute long storyboard that embarks upon an epic, Sci-Fi and RPG inspired adventure.

Becoming something of a household name among the storyboarding scene for their years of excellence, **-Tochi**'s work never fails to attract or impress, making them a very deserving recipient of the Elite Storyboarder title.

Here's a few highlights from them:

> - [Feryquitous - \(S\).0ngs//---::compilation.\[TQR-f3\]](https://osu.ppy.sh/beatmapsets/1124097)
> - [Camellia - chrono diver -fragment- (Camellia's "crossroads of chrono" remix)](https://osu.ppy.sh/beatmapsets/419600)
> - [PSYQUI - Hype feat. Such (lapix Remix)](https://osu.ppy.sh/beatmapsets/1009824)
> - [Function Phantom - Paradox](https://osu.ppy.sh/beatmapsets/824125)

Reviewer 2: ```suggestion
- [Function Phantom - Paradox](https://osu.ppy.sh/beatmapsets/824125)
- [Camellia - chrono diver -fragment- (Camellia's "crossroads of chrono" remix)](https://osu.ppy.sh/beatmapsets/419600)
- [PSYQUI - Hype feat. Such (lapix Remix)](https://osu.ppy.sh/beatmapsets/1009824)
- [Feryquitous - \(S\).0ngs//---::compilation.\[TQR-f3\]](https://osu.ppy.sh/beatmapsets/1124097)
```
follows video order

Author: applied

[![](/wiki/shared/news/2021-03-18-community-contributors-2020/yf_bmp.jpg)](https://osu.ppy.sh/users/1243669)

<div align="center">
    <video width="95%" controls>
        <source src="https://assets.ppy.sh/media/community-contributors-2020/yf_bmp_showcase.mp4" type="video/mp4" preload="none">
    </video>
</div>

Renowned for their notable and distinctive style, **yf_bmp** has long sought to capture the soul and emotion of the songs that their work compliments, showcasing that storyboarding is not always about flashy effects and particles.

> An excellent example of this is their work in [Bassdrop Freaks](https://osu.ppy.sh/beatmapsets/463479), making ample use of bright colours and quick transitions to reflect the upbeat nature of this Camellia and nanahira classic, and again in [Shiawase no Otoshimono](https://osu.ppy.sh/beatmapsets/559622), where they masterfully utilize gentle motion and motifs to create a stunning homage to the source material.

Reviewer 2: ```suggestion
An excellent example of this is their work in [Bassdrop Freaks](https://osu.ppy.sh/beatmapsets/463479), making ample use of bright colours and quick transitions to reflect the upbeat nature of this Camellia and Nanahira classic, and again in [Shiawase no Otoshimono](https://osu.ppy.sh/beatmapsets/559622), where they masterfully utilize gentle motion and motifs to create a stunning homage to the source material.
```

Author: applied

Not only that, but **yf_bmp** is the developer and maintainer of the [OSB optimizer](https://osu.ppy.sh/community/forums/topics/532042) utility, long lauded by the storyboarding scene for its ability to significantly reduce the otherwise cumbersome sizes of many storyboarding projects.

It is plain for any to see the extraordinary amount of effort and care that goes into **yf_bmp**'s creations, and it should be a surprise to nobody that they now join the other creative titans of osu!'s history as one of the Elite Storyboarders.

Here's a few highlights of their work:

> - [Aimer - RE:I AM](https://osu.ppy.sh/beatmapsets/450885)
> - [Nanahira - Bassdrop Freaks (Long Ver.)](https://osu.ppy.sh/beatmapsets/463479)
> - [Duca - Shiawase no Otoshimono](https://osu.ppy.sh/beatmapsets/559622)
> - [DJ NAGAI feat. a*ru - Benibotan](https://osu.ppy.sh/beatmapsets/548679)
> - [Chata - enn](https://osu.ppy.sh/beatmapsets/406217)

Reviewer 2: ```suggestion
- [Chata - enn](https://osu.ppy.sh/beatmapsets/406217)
- [Nanahira - Bassdrop Freaks (Long Ver.)](https://osu.ppy.sh/beatmapsets/463479)
- [DJ NAGAI feat. a*ru - Benibotan](https://osu.ppy.sh/beatmapsets/548679)
- [Duca - Shiawase no Otoshimono](https://osu.ppy.sh/beatmapsets/559622)
- [Aimer - RE:I AM](https://osu.ppy.sh/beatmapsets/450885)
```
video order
